capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -0,0 +1,256 @@
+              {this.props.text || 'Search for Users'}
+            </Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <Form>
+              <Form.Group controlId='form-search-term'>
+                <Form.Control type='text' placeholder='Search' onChange={(event) => this.handleSearchTermChange(event)} />
+                {this.state.fetchingUsers &&
+                  <div className='text-center'>
+                    <Spinner
+                      as='span'
+                      animation='border'
+                      size='lg'
+                      role='status'
+                      aria-hidden='true'
+                      className='my-5'
+                    />
+                  </div>}
+              </Form.Group>
+
+              {
+                this.state.searchResults &&
+
+                <div>
+
+                  {this.state.searchResults.length > 0 &&
+                    <h3 className='my-4 px-4'>Search Results</h3>
+                  }
+
+                  <Row className='px-3 text-center'>
+                    {(!this.state.resultsFound && this.state.searchTerm !== '') &&
+                      <h4 className='margin-auto py-4'>No Users Found</h4>
+                    }
+
+                    {this.state.searchResults.map(user =>
+
+                      <Col key={user.id} md={4} className="px-2 my-0">
+                        <Card >
+                          <Card.Img variant="top" src={user.profileImageUrl} className='img-fluid rounded-circle w-50 margin-auto mt-3' />
+                          <Card.Body>
+                            <Card.Title>
+                              {user.fullName}
+                            </Card.Title>
+                            <Card.Text className='email-text'>
+                              {user.email}
+                            </Card.Text>
+
+
+                            {this.state.addedUsers.includes(user) ?
+                              <Button className="my-2" variant='danger' onClick={() => this.removeUserFromAddedUsers(user)}>
+                                Remove
+                              </Button>
+                              :
+                              <Button className="my-2" onClick={() => this.addUserToAddedUsers(user)}>
+                                Add
+                              </Button>
+                            }
+                            <br />
+                            {this.state.addedFriends.includes(user) ?
+                              <Button variant='danger' className="mt-2 mb-1" onClick={() => this.removeUserFromAddedFriends(user)}>
+                                Remove Friend
+                              </Button>
+                              :
+                              <Button className="mt-2 mb-1" onClick={() => this.addUserToAddedFriends(user)}>
+                                Add Friend
+                              </Button>}
```

Comment:
You should make the buttons have the same width by using something like w-50

Reply:
I will change the width in the next PR.